dev-libs/double-conversion/double_conversion-3.1.5.recipe

```diff
@@ -63,5 +67,6 @@ INSTALL()
 
 TEST()
 {
-	make check
+	make
```

Comment:
No need for "make" here, you can remove the line, make process has already been done in BUILD()